packages/react-components/src/localization/translated/ar.json

```diff
@@ -10,6 +10,15 @@
     "multipleUsersAbbreviateMany": "... وآخرون يكتبون {users}",
     "delimiter": "و "
   },
+  "sendBox": {
```

Comment:
How are you getting these translations?

Reply:
These are just from Google translate. These are just for testing the look and feel of an RTL language that does not use the Roman alphabet. This helped me for TypingIndicator. The proper translations will be provided by TD build.